main/unbound/unbound.initd

```diff
@@ -10,10 +7,14 @@ description="unbound is a Domain Name Server (DNS) that is used to resolve host
 description_configtest="Run syntax tests for configuration files only."
 description_reload="Kills all children and reloads the configuration."
 
+# Upper case variables are here only for backward compatibility.
+: ${cfgfile:=${UNBOUND_CONFFILE:-/etc/unbound/$RC_SVCNAME.conf}}
+: ${checkcfg:=${UNBOUND_CHECKCONF:-/usr/sbin/unbound-checkconf}}
```

Comment:
Once again, we do not expect, nor support, moving installed binaries to different locations. It doesn’t make any sense to make location of `unbound-checkconf` configurable.